### ISSUE TITLE
Missing translations + suggestions for improvements

### ISSUE THREAD

**Issue by trebmuh** [LMMS/lmms#5666]
LMMS 1.2.2

This windows is missing all the translations:

#### Screenshot
![LMMS-traduction](https://user-images.githubusercontent.com/8705846/91973796-204d6900-ed1d-11ea-863d-753a593ed621.png)

----

Suggestion for the following windows: the translation strings should be cut in 2 with the 2nd part being displayed below in case the window isn't large enough

When the window is opened the first time:

![LMMS trad 2 - 1 sur 3](https://user-images.githubusercontent.com/8705846/91974349-f2b4ef80-ed1d-11ea-856d-27de967edea7.png)

If we enlarge the window:

![LMMS trad 2 - 2 sur 3](https://user-images.githubusercontent.com/8705846/91974368-fb0d2a80-ed1d-11ea-8ecd-e1be310278eb.png)

Suggestion of what should be the window with its initial width:

![LMMS trad 2 - 3 sur 3](https://user-images.githubusercontent.com/8705846/91974428-0f512780-ed1e-11ea-9087-cd5f1df38310.png)

**Comment by DomClark** [comment]
I can't reproduce the missing translations on Windows:

![image](https://user-images.githubusercontent.com/1873225/91977068-f777a480-ed19-11ea-85ad-4cf60c8d20fd.png)

How did you get LMMS? Is this the AppImage or from a package manager?

**Comment by trebmuh** [comment]
This is on linux LibraZiK-3 (which is based on Debian Buster).
The LMMS 1.2.2 package is a LZK one, build from https://github.com/LMMS/lmms/releases/download/v1.2.2/lmms_1.2.2.tar.xz

**Comment by DomClark** [comment]
The file dialog translations are provided by Qt. We bundle Qt and its translations in our releases. LMMS will try to load translations from the location they were at during compilation, or from its locale directory if they were bundled. Pass `-DBUNDLE_QT_TRANSLATIONS=ON` to CMake to bundle translations.

**Comment by trebmuh** [comment]
I've tried to follow your advice, but there is no change unfortunately for me.

Here is the build log in the hope that can ring a bell for you: [log-lmms.txt](https://github.com/LMMS/lmms/files/5171058/log-lmms.txt).

_Please, note that I'm building in a Debian Buster chroot and Qt is the Debian Buster version: 5.11.3_

**Comment by DomClark** [comment]
Does your Qt come with translations? They should always be available for local builds if so, since LMMS will read them straight from the Qt installation. Perhaps @PhysSong can help more here.

**Comment by PhysSong** [comment]
Found it, `qttranslations5-l10n` should be installed at the build time. Or you can remove the `IF(EXISTS "${QT_TRANSLATIONS_DIR}")` check in the root `CMakeLists.txt` by a patch.
@DomClark `BUNDLE_QT_TRANSLATIONS` is not meant for Linux packages where the translation is available with separate packages.

**Comment by trebmuh** [comment]
Nice finding @PhysSong , that was the one missing in the build rules I'm using.
Note that since my package rules are based on the Debian ones, I've upstreamed the change to Debian too: https://salsa.debian.org/multimedia-team/lmms/-/commit/221dc027e1e60bf27cac6b5951def686c1e2c621

Thanks @DomClark and @PhysSong for the help here.
Closing here and will open a new issue for the suggestions for improvements.
